lib/store/actions.js

```diff
@@ -19,6 +19,8 @@ const {
   HEALTHY,
 } = require('./constants');
 
+let watchID;
```

Comment:
Not a fan of leaky state like this, but prepared to live with it for the time being.